tasks/Resources/Provisioning/Disconnect/icsp.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: generic-0
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - registry.rh-ods.com:8443/openshift
```

Comment:
could this become a parameter?

Reply:
For now, we need to use always this repo